cmd/plasmacli/subcmd/query/info.go

```diff
@@ -73,3 +68,19 @@ var infoCmd = &cobra.Command{
 		return nil
 	},
 }
+
+func Info(ctx context.CLIContext, addr ethcmn.Address) ([]store.TxOutput, error) {
```

Comment:
Is this used within this command? I think it should be removed. The two line query/marshal should just be c/p'ed in places that need info. It's more clear/explicit that way